src/components/Card.js

```diff
@@ -13,36 +25,86 @@ export default class Card {
     return cardTemplate;
   }
 
+  // old code, use this first
+
+  // _toggleLike = () => {
+  //   // Do some API function to handle like and unlike
+  //   // After API call, update this._likes
+  //   // Update like count and like button state
+  //   this._likeButton.classList.toggle("card__like-button-active");
+  // };
+
+  // updated code, not sure.
   _toggleLike = () => {
-    this._likeButton.classList.toggle("card__like-button-active");
+    // check if an of the likes belong to you
+    const isLiked = this._likeButton.classList.contains(
+      "card__like-button-active"
+    );
+    this._isLiked = isLiked;
```

Comment:
Please remove this, `isLiked` value should be calculated (use `card.isLiked()` in index.js)